.pre-commit-hooks.yaml

```diff
@@ -6,4 +6,5 @@
   language: python
   language_version: python3
   require_serial: true
+  verbose: true
```

Comment:
I'm temporarily removing this from https://pre-commit.com to unbreak the build -- `verbose: true` is a debug-only flag and should not be set in `.pre-commit-hooks.yaml`

Reply:
Apologies for (unintentionally) breaking the build. I removed `verbose: true` from the `.pre-commit-hooks.yaml` and released `v0.3.2`.